app/frontend/components/pages/login/loginPageSidebar.js

```diff
@@ -0,0 +1,183 @@
+const {h, Component} = require('preact')
+const connect = require('unistore/preact').connect
+
+const Alert = require('../../../components/common/alert')
+
+class LoginPageSidebar extends Component {
+  render({authMethod}) {
+    const initialContent = () =>
```

Comment:
Please, define these components outside of the render method for `LoginPageSidebar`. They should be components define still in this same file but render method for `LoginPageSidebar` should be:

```js
return h(
      'aside',
      {class: 'sidebar'},
      authMethod === '' && h(InitialContent),
      authMethod === 'mnemonic' && h(MnemonicContent),
      authMethod === 'trezor' && h(WalletContent),
      authMethod === 'file' && h(FileContent)
    )
```